tools/cldr-code/src/main/java/org/unicode/cldr/util/MatchValue.java

```diff
@@ -616,7 +631,14 @@ public static SetMatchValue of(String key) {
 
         @Override
         public  boolean is(String items) {
-            return and(subtest,SPACE_SPLITTER.split(items));
+            Iterable<String> splitItems = SPACE_SPLITTER.split(items);
```

Comment:
Looks good. Should add quick unit test.
Also, can be pithier. Split to list, then create set from it. If the sizes are not the same, fail.

Reply:
> Should add quick unit test.

TestAttributeValues already hits this, and is verified to fail on the `"… fil fil …"` case for grammatical features.

> Also, can be pithier… If the sizes are not the same, fail

good point, will fix